docs/uml-class-diagrams/display02/test/DisplayCommInterfaceImplementationTests/DisplayCommInterfaceImplementationTests.puml

```diff
@@ -2,8 +2,11 @@
 
 package edu.oakland.test.display02 {
     class DisplayCommInterfaceImplementationTests {
+        + void isRfidRecieved()
```

Comment:
You know that this will be true already, or the class would not have been called. You can thus remove this test method (and save yourself some work).
```suggestion
```